tools/schema/datatypes_minimal.txt

```diff
@@ -24,6 +24,7 @@ DataValue
 Variant
 DiagnosticInfo
 NodeClass
+ReferenceNode
```

Comment:
All these DataTypes should not be added to datatypes_minimal.txt but to datatypes_pubsub.txt.
Check individually if they are really needed for PubSub. Otherwise we add dead weight to the binaries.